# BEGINNING  - The Hypergeometrical Universe Theory
# Derivation of Laws of Nature

BEGINNING  - The Hypergeometrical Universe Theory - Derivation of Laws of Nature

###  Interaction Model in a 4D Spatial Manifold

#### Approximation of the Dilaton Field

Step 1: Defining the Field $\Phi_1$ 

The total dilaton field from body 1 is given by:
$\Phi_1 = \cos(k_1 \cdot x) $ 
where  $k_1$  is the wave vector and  x  is the position vector.

Step 2: Calculating the First Derivative of$\Phi_1$

Using the approximation that $\Phi_1 = \cos(k_1 \cdot x) $ , we calculate the first derivative:
$\frac{d\Phi_1}{dx} = (k_1 \cdot \hat{x}) \sin(k_1 \cdot x )= (k_1 \cdot \hat{x})(k_1 \cdot x )$ 

Step 3: Calculating the Second Derivative of $\Phi_1$

Next, we find the second derivative to eliminate any lingering component with  $x^2$ since $k_1$ might depend upon x:
$\frac{d^2\Phi_1}{dx^2} = (k_1 \cdot \hat{x})^2 \cos(k_1 \cdot x)$ 

Step 4: Extracting the Coefficient of  x **

We set  x = 0  to extract the coefficient of  x :
$\left. \frac{d^2\Phi_1}{dx^2} \right|_{x=0} = (k_1 \cdot \hat{x})^2$ 

Here, we approximate  x  as  $x = x_ \cdot \hat{x} $, where  x_  is the magnitude and  $\hat{x}$  is the unit vector in the direction of  x .

Step 5: Calculating the First Derivative of $\Phi_2$

Similarly, we calculate the first derivative of the dilaton field from body 1 to body 2,
$\Phi_2: \frac{d\Phi_2}{dx} = (k_2 \cdot \hat{x}) \sin(k_2 \cdot x)$ 

Step 6: Solving for  $x_$

To align the fields in phase, we solve for  x_ :
 $ x_ = \frac{1}{(k_1 \cdot \hat{x})^2} \cdot \frac{d\Phi_2}{dx} $

This approximation and calculation show how the position  x_  of body 1 is adjusted to ensure that it dilates space in phase with the field from body 2.



In [1]:
import sympy as sp


def cleanForce(Force):
    Force_latex = sp.latex(Force)
    Force_latex = Force_latex.replace("V_{1 hat}", "\\hat{V}_1")
    Force_latex = Force_latex.replace("V_{2 hat}", "\\hat{V}_2")
    Force_latex = Force_latex.replace("R_{0 hat}", "\\hat{R}_0")
    Force_latex = Force_latex.replace("r_{hat}", "\\hat{r}")
    # Force_latex = Force_latex.replace("+ 0 \hat{r}", "")
    Force_latex = Force_latex.replace("^{T}", "")
    print(Force_latex)


# Define symbols
pi, m_0, x, Q, lambda1, R0, c, N, v1, v2, gamma_v1, gamma_v2, P1, P2, delta, R_4D, P1_hat, P2_hat, G0, kg, h  = sp.symbols(
    'pi m_0 x Q lambda1 R0 c N v1 v2 gamma_v1 gamma_v2 P1 P2  delta R_4D P1_hat P2_hat G0 kg h', 
    real=True, positive=True)
alpha = sp.symbols('alpha', real=True, positive=True)

# Define MatrixSymbols for arbitrary unit vectors
V1_hat = sp.MatrixSymbol('V1_hat', 3, 1)  # 3x1 Matrix Symbol for Unit vector along V1
V2_hat = sp.MatrixSymbol('V2_hat', 3, 1)  # 3x1 Matrix Symbol for Unit vector along V2
R0_hat = sp.MatrixSymbol('R0_hat', 3, 1)  # 3x1 Matrix Symbol for Unit vector along R0
r_hat = sp.MatrixSymbol('r_hat', 3, 1)    # 3x1 Matrix Symbol for general unit vector for position adjustments

# Identity matrix for space dimension
One = sp.MatrixSymbol('One', 3, 3) # Correct usage of Identity Matrix

# Velocity vectors
V1 = v1 * V1_hat
V2 = v2 * V2_hat
R0_vect = R0 * R0_hat

# Position vectors
r1 = V1 * R0 / c + x * r_hat
r2 = V1 * R0 / c - R0_vect + x * r_hat

# Projection matrices and Lorentz transformation matrices
# 1+(GAMMA-1)P = (1-P) + GAMMA*P
# P IS THE PROJECTOR TO THE V1 AND V2 DIRECTIONS 

M1 = One + (gamma_v1 - 1) * V1 * V1.T / (v1**2)
M2 = One + (gamma_v2 - 1) * V2 * V2.T / (v2**2)

# Wave vectors
k1 = (2 * pi / lambda1) * r1.T* M1  / P1
k2 = (2 * pi / lambda1) * r2.T * M2 / P2

# P1 is the modulus of r1 or |r1| or |v1 R0/c|
# P2 is the modulus of r2 or |r2| or R0*sqrt (v1**2/c**2 + 1 - 2 dr/dt)

# Fields definitions
k1path = ( k1 * r1 )
Phi1 = sp.cos(k1path)
first_derivative = sp.diff(k1path, x)
second_derivative = sp.diff(first_derivative, x)
# Took the second derivative to extract the coefficient of x
k1path_diff = second_derivative
k1path=k1path.subs(x,0)
k1path

(2*pi/(P1*lambda1))*((R0*v1/c)*V1_hat.T + 0*r_hat.T)*((gamma_v1 - 1)*V1_hat*V1_hat.T + One)*((R0*v1/c)*V1_hat + 0*r_hat)

# Calculate $ \frac{d\Phi_1}{dr} $

In [2]:
# Derivatives - Here we used diff(cos(k.x) = -ksin(kx) = -k**2x
Phi1_diff = k1path.T*k1path_diff
Phi1_diff

(8*R0**2*pi**2*v1**2/(P1**2*c**2*lambda1**2))*V1_hat.T*((gamma_v1 - 1)*V1_hat*V1_hat.T + One.T)*V1_hat*r_hat.T*((gamma_v1 - 1)*V1_hat*V1_hat.T + One)*r_hat

In [3]:
cleanForce(Phi1_diff)

\frac{8 R_{0}^{2} \pi^{2} v_{1}^{2}}{P_{1}^{2} c^{2} \lambda_{1}^{2}} \hat{V}_1 \left(\left(\gamma_{v1} - 1\right) \hat{V}_1 \hat{V}_1 + One\right) \hat{V}_1 \hat{r} \left(\left(\gamma_{v1} - 1\right) \hat{V}_1 \hat{V}_1 + One\right) \hat{r}


# Calculate $ \frac{d\Phi_2}{dr} $

In [4]:
k2path = ( k2 * r2 )
k2path =Q * k2path/(2*pi)
k2path_diff = sp.diff(k2path,x,2)
Phi2 = N / (1 + k2path[0].subs(x,0))
# Derivatives
Phi2_diff = -N *k2path_diff / (k2path.subs(x,0))**2 
Phi2_diff

(-2*N*Q/(P2*lambda1))*r_hat.T*((gamma_v2 - 1)*V2_hat*V2_hat.T + One)*r_hat*((Q/(P2*lambda1))*((-R0)*R0_hat.T + (R0*v1/c)*V1_hat.T)*((gamma_v2 - 1)*V2_hat*V2_hat.T + One)*((-R0)*R0_hat + (R0*v1/c)*V1_hat))**(-2)

In [5]:
cleanForce(Phi2_diff)

- \frac{2 N Q}{P_{2} \lambda_{1}} \hat{r} \left(\left(\gamma_{v2} - 1\right) \hat{V}_2 \hat{V}_2 + One\right) \hat{r} \left(\frac{Q}{P_{2} \lambda_{1}} \left(- R_{0} \hat{R}_0 + \frac{R_{0} v_{1}}{c} \hat{V}_1\right) \left(\left(\gamma_{v2} - 1\right) \hat{V}_2 \hat{V}_2 + One\right) \left(- R_{0} \hat{R}_0 + \frac{R_{0} v_{1}}{c} \hat{V}_1\right)\right)^{-2}


# Calculate Force
$ \vec{F} = m_0\frac{dv}{dt} = m_0 c^2 \frac{d(\frac{v}{c})}{dr_2} \frac{dr_2}{dR}=  m_0 c^2 \frac{dtanh(\alpha)}{dr_1}\frac{dr_1}{dR}$ 

Where
$ tanh(\alpha)= \frac{v}{ic} $
 
$ \vec{r_2} = \vec{V_1} * R_0 / c - \vec{R_0}  $

$ \frac{dr_2}{dR} =  \sqrt{ 1 + \frac{v_1^2}{c^2} - \frac{2}{c}\frac{dr_1}{dt}}$

$ \vec{F} = m_0 c^2 (1-\frac{v_1^2}{c^2}) \frac{d(\frac{v}{c})}{cdt} \frac{dr_1}{dR}$

$ \vec{F} = m_0 c^2 (1+tanh^2(\alpha))\frac{d\alpha}{dr_1} \frac{dr_1}{dR}$

$ \alpha = \frac{x}{\lambda_1} $

$ \frac{d\alpha}{dr} =\frac{\alpha}{\lambda_1} $

$ \vec{F} = m_0 c^2 (1-\frac{v_1^2}{c^2})\frac{\alpha}{\lambda_1^2}\frac{dr_1}{dR} $


In [6]:
# Display results
dr1dr = P2_hat
x_result = Phi2_diff/Phi1_diff
Force_EM = m_0*c**2*(1-v1**2/c**2)*x_result/lambda1**2
Force_EM = Force_EM.subs(Q, 1/(2*pi*alpha))
Force_EM = Force_EM.subs(x,0).simplify()*dr1dr
Force_EM

(N*P2_hat*m_0*(-c**2 + v1**2)/(P2*alpha*lambda1**3*pi))*r_hat.T*((gamma_v2 - 1)*V2_hat*V2_hat.T + One)*r_hat*((1/(2*P2*alpha*lambda1*pi))*((-R0)*R0_hat.T + (R0*v1/c)*V1_hat.T)*((gamma_v2 - 1)*V2_hat*V2_hat.T + One)*((-R0)*R0_hat + (R0*v1/c)*V1_hat))**(-2)*((8*R0**2*pi**2*v1**2/(P1**2*c**2*lambda1**2))*V1_hat.T*((gamma_v1 - 1)*V1_hat*V1_hat.T + One.T)*V1_hat*r_hat.T*((gamma_v1 - 1)*V1_hat*V1_hat.T + One)*r_hat)**(-1)

In [7]:
print(Force_EM)


(N*P2_hat*m_0*(-c**2 + v1**2)/(P2*alpha*lambda1**3*pi))*r_hat.T*((gamma_v2 - 1)*V2_hat*V2_hat.T + One)*r_hat*((1/(2*P2*alpha*lambda1*pi))*((-R0)*R0_hat.T + (R0*v1/c)*V1_hat.T)*((gamma_v2 - 1)*V2_hat*V2_hat.T + One)*((-R0)*R0_hat + (R0*v1/c)*V1_hat))**(-2)*((8*R0**2*pi**2*v1**2/(P1**2*c**2*lambda1**2))*V1_hat.T*((gamma_v1 - 1)*V1_hat*V1_hat.T + One.T)*V1_hat*r_hat.T*((gamma_v1 - 1)*V1_hat*V1_hat.T + One)*r_hat)**(-1)


In [8]:
cleanForce(Force_EM)

\frac{N P_{2 hat} m_{0} \left(- c^{2} + v_{1}^{2}\right)}{P_{2} \alpha \lambda_{1}^{3} \pi} \hat{r} \left(\left(\gamma_{v2} - 1\right) \hat{V}_2 \hat{V}_2 + One\right) \hat{r} \left(\frac{1}{2 P_{2} \alpha \lambda_{1} \pi} \left(- R_{0} \hat{R}_0 + \frac{R_{0} v_{1}}{c} \hat{V}_1\right) \left(\left(\gamma_{v2} - 1\right) \hat{V}_2 \hat{V}_2 + One\right) \left(- R_{0} \hat{R}_0 + \frac{R_{0} v_{1}}{c} \hat{V}_1\right)\right)^{-2} \left(\frac{8 R_{0}^{2} \pi^{2} v_{1}^{2}}{P_{1}^{2} c^{2} \lambda_{1}^{2}} \hat{V}_1 \left(\left(\gamma_{v1} - 1\right) \hat{V}_1 \hat{V}_1 + One\right) \hat{V}_1 \hat{r} \left(\left(\gamma_{v1} - 1\right) \hat{V}_1 \hat{V}_1 + One\right) \hat{r}\right)^{-1}


In [9]:
latex_str = sp.latex(Force_EM, mode='plain')
# Write to a file
with open('./Drawing_For_Publications/ForceEM.tex', 'w') as file:
    file.write(latex_str)

# Calculate Gravitational Force
$ \vec{F} = m_0\frac{dv}{dt} = m_0 c^2 \frac{d(\frac{v}{c})}{dr}=  m_0 c^2 \frac{dtanh(\alpha)}{dr_1} \frac{dr_1}{dr}$ 

Where
 $ tanh(\alpha)= \frac{v}{ic} $

$ \vec{F} = m_0 c^2 (1-\frac{v_1^2}{c^2}) \frac{d(\frac{v}{c})}{cdt} $

$ \vec{F} = m_0 c^2 (1-tanh^2(\alpha))\frac{d\alpha}{dr_1} \frac{dr_1}{dr} $

$ \alpha = \frac{\delta x}{R_{4D}} $

$ \frac{d\alpha}{dr_1} =\frac{\delta x}{R_{4D}\lambda_1} $

$ \vec{F} = m_0 c^2 (1-\frac{v_1^2}{c^2})\frac{\delta x}{R_{4D}\lambda_1} \frac{dr_1}{dr}$

In [10]:
# Display results
dr1dr = P2_hat
x_result = Phi2_diff/Phi1_diff
Force_G = m_0*c**2*(1-v1**2/c**2)*delta * x_result/lambda1/R_4D*(N/kg**2)
Force_G = Force_G.subs(Q, 1/(2*pi*alpha))
Force_G = Force_G.subs(x,0).simplify()*dr1dr
Force_G

(N**2*P2_hat*delta*m_0*(-c**2 + v1**2)/(P2*R_4D*alpha*kg**2*lambda1**2*pi))*r_hat.T*((gamma_v2 - 1)*V2_hat*V2_hat.T + One)*r_hat*((1/(2*P2*alpha*lambda1*pi))*((-R0)*R0_hat.T + (R0*v1/c)*V1_hat.T)*((gamma_v2 - 1)*V2_hat*V2_hat.T + One)*((-R0)*R0_hat + (R0*v1/c)*V1_hat))**(-2)*((8*R0**2*pi**2*v1**2/(P1**2*c**2*lambda1**2))*V1_hat.T*((gamma_v1 - 1)*V1_hat*V1_hat.T + One.T)*V1_hat*r_hat.T*((gamma_v1 - 1)*V1_hat*V1_hat.T + One)*r_hat)**(-1)

In [11]:
Force_G = Force_G.subs(gamma_v2,1).subs(delta, G0*(2*pi*(kg**2)*R_4D)/(N**2*m_0*c**2*alpha*lambda1**2))
Force_G = Force_G.simplify()
print(Force_G)

(2*G0*P2_hat*(-c**2 + v1**2)/(P2*alpha**2*c**2*lambda1**4))*r_hat.T*(One + 0*V2_hat*V2_hat.T)*r_hat*((1/(2*P2*alpha*lambda1*pi))*((-R0)*R0_hat.T + (R0*v1/c)*V1_hat.T)*(One + 0*V2_hat*V2_hat.T)*((-R0)*R0_hat + (R0*v1/c)*V1_hat))**(-2)*((8*R0**2*pi**2*v1**2/(P1**2*c**2*lambda1**2))*V1_hat.T*((gamma_v1 - 1)*V1_hat*V1_hat.T + One.T)*V1_hat*r_hat.T*((gamma_v1 - 1)*V1_hat*V1_hat.T + One)*r_hat)**(-1)


In [12]:
cleanForce(Force_G)

\frac{2 G_{0} P_{2 hat} \left(- c^{2} + v_{1}^{2}\right)}{P_{2} \alpha^{2} c^{2} \lambda_{1}^{4}} \hat{r} \left(One + 0 \hat{V}_2 \hat{V}_2\right) \hat{r} \left(\frac{1}{2 P_{2} \alpha \lambda_{1} \pi} \left(- R_{0} \hat{R}_0 + \frac{R_{0} v_{1}}{c} \hat{V}_1\right) \left(One + 0 \hat{V}_2 \hat{V}_2\right) \left(- R_{0} \hat{R}_0 + \frac{R_{0} v_{1}}{c} \hat{V}_1\right)\right)^{-2} \left(\frac{8 R_{0}^{2} \pi^{2} v_{1}^{2}}{P_{1}^{2} c^{2} \lambda_{1}^{2}} \hat{V}_1 \left(\left(\gamma_{v1} - 1\right) \hat{V}_1 \hat{V}_1 + One\right) \hat{V}_1 \hat{r} \left(\left(\gamma_{v1} - 1\right) \hat{V}_1 \hat{V}_1 + One\right) \hat{r}\right)^{-1}


In [13]:
from astropy import units as uu, constants as cc
import numpy as np

# NOTICE THAT I AM USING THE ACTUAL MASS OF A HYDROGEN ATOM HERE AS OPPOSED TO HALF-MASS AS I DID FOR THE 
# ELECTROMAGNETIC FUNDAMENTAL DILATOR

m_H=cc.m_p+cc.m_e
N_= (uu.kg/m_H).si
# So 1kg_4D is equivalent to 5.9753831E26 Fundamental Dilators 
# or 5.9753831E26 electron charges or 95736192 Coulombs
R_4D= (14.04E9*uu.lyr).si
lambda_1 = (cc.h/m_H/cc.c).si
lambda_1
dt = lambda_1/cc.c
G_HU =  (N_*cc.h*cc.c*cc.alpha*lambda_1/R_4D/(2 *np.pi))

G_HU =(N_**2* cc.h*cc.c*cc.alpha*lambda_1)/(2*np.pi*R_4D*uu.kg**2) 
G_delta = cc.G/G_HU.si
G_delta

<Quantity 81489.69936511>

In [14]:
latex_str = sp.latex(Force_G, mode='plain')
# Write to a file
with open('./Drawing_For_Publications/ForceG.tex', 'w') as file:
    file.write(latex_str)

In [15]:
# # Expand in Taylor series with respect to v1 and then v2
# taylor_v1 = Force.series(v1, n=1).removeO()  # Expand to v1^2 and remove the Big O notation
# final_taylor = taylor_v1.series(v2, n=1).removeO()  # Now expand to v2^2

# # Simplify the expression (this step is optional but recommended for cleaner output)
# final_taylor_simplified = sp.simplify(final_taylor)

# # Display the final expanded Taylor series
# print(final_taylor_simplified)


# NOTICE THAT I AM USING THE ACTUAL MASS OF A HYDROGEN ATOM HERE AS OPPOSED TO HALF-MASS AS I DID FOR THE ELECTROMAGNETIC FUNDAMENTAL DILATOR

In [16]:
m0_,  c, pi, alpha_, N_ ,R0_ ,lambda1_, h = sp.symbols("m0 c pi alpha N R0 lambda1 h")
G_HU_ =  (N_**2*h*c*alpha*lambda1_/R0_/(2 *pi))
G_HU_

N**2*alpha*c*h*lambda1/(2*R0*pi)

# Notice that for the Gravitational Fundamental Dilator, the 4D mass is 1 Hydrogen atom
# Notice that for electromagnetism, the Fundamental Dilator has a 4D mass of HALF Hydrogen Atom

# Natural Frequency of Gravitational Waves is given by this simple model
$$ F = mc^2 \frac{dtan(\alpha_0)}{dr}=mc^2 \frac{\frac{\delta x}{R_0}}{\lambda_1} = m (2 \pi f)^2 x $$

$$ f = \frac{c}{2 \pi} \sqrt{ \frac{\delta}{\lambda_1R_0}    } $$

In [17]:
frequency = cc.c/(2*np.pi) *np.sqrt( G_delta/R_4D/lambda_1)
frequency.si

<Quantity 32519.64249857 1 / s>

# The Natural Frequency of Gravitational Waves is almost 32519.642 Hertz
# Since the Q of Space is extremely high, the resonant excitation of Gravitational WaveS will require some precise mechanics

In [18]:
 G_delta,  R0, lambda_1

(<Quantity 81489.69936511>, R0, <Quantity 1.32069058e-15 m>)

# WHAT CAN ONE DO WITH GRAVITATIONAL WAVES?
# ONE CAN MAKE A GRAVITATIONAL ACCELERATOR WHERE NEUTRAL MATTER IS ACCELERATED TO THE SPEED OF LIGHT INSTANTANEOUSLY
# THIS IS HOW ONE CREATES A REAL WARP DRIVE

One of the most famous observations of neutron star orbits via gravitational waves is from the event GW170817. This event was the first-ever detection of gravitational waves from a binary neutron star merger, observed by LIGO and its sister observatory, Virgo, on August 17, 2017. 

During the final minutes before the merger, the neutron stars were orbiting each other at an incredibly high speed. To give you a sense of how fast they were orbiting, let's use the fact that the gravitational wave frequency is twice the orbital frequency of the binary system due to the quadrupole nature of gravitational radiation.

At the point of the last observable orbit before merger, the gravitational wave frequency was about 1500 Hz. Therefore, the orbital frequency $ ( f_{\text{orb}} ) $ of the neutron stars was approximately half of that, which is 750 Hz. This means the neutron stars were completing 750 orbits around each other every second just before they merged.

Using the formula for the orbital frequency of two objects in a circular orbit:
$$  f_{\text{orb}} = \frac{1}{2\pi} \sqrt{\frac{G(M_1 + M_2)}{r^3}}  $$

where \( G \) is the gravitational constant, \( M_1 \) and \( M_2 \) are the masses of the two neutron stars, and $ r $ is the distance between their centers, we could technically work backwards to find $ r $ if we knew the masses. However, it's simpler to state that, as they approached merger, their separation was just a few tens of kilometers, and they were moving at a significant fraction of the speed of light.

To summarize, the neutron stars in the event GW170817 were orbiting each other at an orbital frequency of approximately 750 Hz, which means they were orbiting each other 750 times per second just before merging. This is an astonishingly rapid orbit, showcasing the extreme conditions under which these cosmic events occur.

Yes, LIGO (Laser Interferometer Gravitational-Wave Observatory) and other gravitational wave detectors like Virgo have to contend with a wide range of noise sources that can affect their ability to detect gravitational waves. These noises come from various sources, including seismic activity, thermal vibrations, and human-made noise. Human-made or "anthropogenic" noise can include frequencies that are harmonics of electrical supply systems (like 60 Hz in the United States and 50 Hz in many other countries) and its multiples, which would include 240 Hz.

LIGO's sensitivity to gravitational waves is highest in a band roughly between 10 Hz and a few kHz, which means it must be capable of distinguishing gravitational wave signals from noise across this range. To achieve this, LIGO uses a combination of physical isolation techniques for its mirrors and laser equipment, sophisticated vibration damping systems, and noise-cancellation techniques akin to noise-cancelling headphones, but much more advanced.

Specific issues with noise at 240 Hz could arise from electronic equipment, including motors, transformers, and other devices that operate with frequencies that are harmonics of the main electrical supply frequency. LIGO scientists and engineers work continuously to identify the sources of such noise and mitigate them, either by physically removing or isolating the source, using active noise cancellation, or by filtering out these frequencies during data analysis.

While I don't have specific instances where 240 Hz noise was highlighted as a problem for LIGO, it's reasonable to assume that any persistent, specific noise frequency within LIGO's sensitivity range would need to be addressed as part of their ongoing efforts to improve the observatory's sensitivity and reliability in detecting gravitational waves.

In [19]:
from astropy import units as uu, constants as cc
m_H = cc.m_e + cc.m_p
lambda1 = (cc.h/(m_H*cc.c)).si
R0_4D = (14.04E9*uu.lyr).si
ratio = lambda1*81489.699/R0_4D # 1.62E-36
# Ratio between Gravitational and EM Forces, considering that the 4D mass of EM Fundamental Dilator is 0.5 the mass of a hydrogen atom
# The Hydrogen Atom is the Gravitational Fundamental Dilator
ratio, (cc.G*m_H**2/(cc.e.si**2/(4*np.pi*cc.eps0))).si


(<Quantity 8.10236899e-37>, <Quantity 8.10236903e-37>)

In [20]:
#Mass of Milky Way 1.5 trillion Solar Masses
NumberOfDilators = (1.5E12*cc.M_sun)/m_H
NumberOfDilators

<Quantity 1.78222661e+69>

In [21]:
NN = (uu.kg/m_H).si
G_HU = (m_H*cc.c**2*NN**2*cc.alpha*lambda1**2/(2*np.pi*R0_4D)/uu.kg**2).to(uu.N*uu.m**2/uu.kg**2)
(cc.G/G_HU).si, G_HU

(<Quantity 81489.69936511>, <Quantity 8.19036032e-16 m2 N / kg2>)